recipes/crashpad/all/conanfile.py

```diff
@@ -1,20 +1,28 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
-from contextlib import contextmanager
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rename, replace_in_file, rm, save, chdir
+from conan.tools.scm import Version
+from conan.tools.build import check_min_cppstd
+from conan.tools.apple import XCRun, is_apple_os
+from conan.tools.env import Environment, VirtualBuildEnv, VirtualRunEnv
+from conan.tools.microsoft import VCVars, is_msvc
+from conan.tools.gnu import AutotoolsDeps, AutotoolsToolchain
+
+
 import os
 import textwrap
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.64.0"
 
 
 class CrashpadConan(ConanFile):
     name = "crashpad"
     description = "Crashpad is a crash-reporting system."
     url = "https://github.com/conan-io/conan-center-index"
-    topics = ("conan", "crashpad", "crash", "error", "stacktrace", "collecting", "reporting")
+    topics = ("crashpad", "crash", "error", "stacktrace", "collecting", "reporting")
     license = "Apache-2.0"
     homepage = "https://chromium.googlesource.com/crashpad/crashpad/+/master/README.md"
-    provides = "crashpad", "mini_chromium"
+    provides = "mini_chromium"
```

Comment:
Missing `package_type`, but I actually don't know what would fit best here. I guess library even though it also exposes bins